levelZeroLib/src/ze_log.h

```diff
@@ -55,4 +55,9 @@
         << std::endl;                                             \
     }
 
+#define ASSIGN_IF_NOT_MINUS_ONE(value, cpp_target, type) \
+    if (value != -1) { \
+        cpp_target = reinterpret_cast<type>(value); \
+    }
+
```

Comment:
Strange location for this Macro. I will open a new file for this type of macros.